src/dask_awkward/awkward.yaml

```diff
@@ -8,10 +8,14 @@ awkward:
   # typetracer, which is an empty awkward array.
   compute-unknown-meta: True
 
-  # This option is executing the column projection optimization. It's
-  # currently implemented as a brute force search for columns that are
-  # required for the task graph to complete without raising an
-  # IndexError (each column is held out and the metadata task graph is
-  # computed as a requirement test); because of this search method the
-  # optimization can be slow, so it is configurable.
-  column-projection-optimization: True
+  # column-projection-optimization currently has 3 possible values
+  # - "brute-force": a brute force search for columns that are
+  #   required for the task graph to complete without raising an
+  #   IndexError (each column is held out and the metadata task graph
+  #   is computed as a requirement test)
+  # - "simple-getitem": checks for calls to getitem in the task graph.
+  #   This optimization is determined faster than the brute-force
+  #   method, but it may not be as complete w.r.t. eliminating
+  #   unncessary reads.
+  # - null, false, or "none": To turn off the optimization
+  column-projection-optimization: "simple-getitem"
```

Comment:
May as well add "chained" right now, which raises a NotImplemented until it's done.